src/index.js

```diff
@@ -22,32 +51,37 @@ export default function css(options = {}) {
         }
       }
 
-      // Track the order that each stylesheet is imported.
-      if (!order.includes(id)) {
-        order.push(id)
-      }
-
       // Keep track of every stylesheet
       // Check if it changed since last render
+      // NOTE: If we are in transform block, we can assume styles[id] !== code, right?
```

Comment:
I think that this section of code could just be
```js
styles[id] = code
hasChanged = true
```
I don't think the if statement is needed here

Reply:
Alright